xAPI.md

```diff
@@ -3406,8 +3412,11 @@ Example endpoint: http://example.com/xAPI/activities/state
 Stores, changes, fetches, or deletes the document specified by the given stateId that 
 exists in the context of the specified Activity, Agent, and registration (if specified).  
 
-Returns (PUT | POST | DELETE): ```204 No Content```  
-Returns (GET): ```200 OK```, State Content  
+**Content (PUT | POST):** The Document object to be stored or updated.  
```

Comment:
Isn't necessarily an object (with PUT); we generally just call them documents, so propose we continue to do so (lowercase).